Gruntfile.js

```diff
@@ -493,19 +493,19 @@ module.exports = function(grunt) {
       },
       'setup-test-database': {
         cmd: [
-          `docker run -d -p 4984:5984 -p 4986:5986 --name e2e-couchdb --mount type=tmpfs,destination=/opt/couchdb/data apache/couchdb:latest`,
-          'sleep 1',
+          `docker run -d -p 4984:5984 -p 4986:5986 --name e2e-couchdb --mount type=tmpfs,destination=/opt/couchdb/data couchdb:2`,
```

Comment:
You should drop the tmpfs stuff since it doesn't work on MacOS. We can add it back just for linux later once we determine it's worth it

Reply:
cool I've removed. I tested on mac os but not windows. It seems the run command works but disregards the tmpfs flag.